packages/@azure/servicebus/data-plane/README.md

```diff
@@ -1,22 +1,30 @@
-# @azure/service-bus
+# Azure Service Bus client library for Typescript and Javascript
 
-This library provides a convenient way to interact with [Azure Service Bus](https://azure.microsoft.com/en-us/services/service-bus/).
+Microsoft Azure Service Bus is a highly-reliable cloud messaging service between applications and services.
+
+Use the client library for Azure Service Bus to 
+- Send messages to a Queue or Topic
```

Comment:
I think the list is unnecessary here. "Use the client library forAzure Service Bus to send and receive messages on a Queue or Topic."

Only concern is that queue and topic are service-specific terms that have not been defined.

Reply:
I want to keep the list.

It is much easier to understand were can you send and where can you receive from. Putting it all in 1 sentence brings in a lot of ands and ors

Also, now that I have added "in your Nodejs application", it is even more beneficial to not to colb into the same sentence

Reply:
Node.js or Node.